RELEASE_NOTES.md

```diff
@@ -2,15 +2,27 @@
 
 ## Summary
 
-<!-- Here goes a general summary of what this release is about -->
+
 
 ## Upgrading
 
 <!-- Here goes notes on how to upgrade from previous versions, including deprecations and what they should be replaced with -->
 
 ### Cookiecutter template
 
-<!-- Here upgrade steps for cookiecutter specifically -->
+- A new script for migrating to new templates (instead of regenerating all files) is provided. It can't handle the upgrade 100% automatically, but should make the migration process much easier and less error prone.
+
+  To run it, the simplest way is to fetch it from GitHub and run it directly:
+
+  ```console
+  curl -sSL https://raw.githubusercontent.com/frequenz-floss/frequenz-repo-config-python/v0.10.0/cookiecutter/migrate.sh | sh
```

Comment:
Could we also provide something like `python -m frequenz.repo-config.migrate v0.10.0` or so? Might be more accessible

Reply:
Yes, that's the end goal, I even created a repo label `part:cli` but the cli tool never came because of time issues. I prefer not to start now because I want to split this repo soon :tm: (to smaller libraries, like https://github.com/frequenz-floss/setuptools-betterproto/) and a repository for the cookiecutter templates. The migration tool should definitely live in the cookiecutter template repo, but other commands should live elsewhere (like the [version calculation tools])(https://github.com/frequenz-floss/frequenz-repo-config-python/tree/v0.x.x/src/frequenz/repo/config/cli/version/mike) used by the CI).

So for now I preferred to go with something very simple, to see how it works and not to build a huge thing that will need to be teared apart soon.